designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -157,7 +160,22 @@ class ProcessesResources(
               complete {
                 processRepository.fetchLatestProcessDetailsForProcessId[CanonicalProcess](processId.id).map[ToResponseMarshallable] {
                   case Some(process) if skipValidateAndResolve => toProcessDetails(enrichDetailsWithProcessState(process))
-                  case Some(process) => validateAndReverseResolve(enrichDetailsWithProcessState(process))
+                  case Some(process) => {
+                    val processTypeData = typeToConfig.forType(process.processingType)
```

Comment:
The same as in `NodeResources` case - keep resource classes as small as it possible, here you should have only http binding logic like conversions to http status codes etc.